src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_consumer_system_v2/test_compressor_system_v2_results/consumer_system_v2.json

```diff
@@ -77,23 +147,23 @@
                         "2022-01-01 00:00:00",
                         "2026-01-01 00:00:00"
                     ],
-                    "typ": "CALENDAR_DAY",
-                    "unit": "Sm3/d",
+                    "typ": "STREAM_DAY",
+                    "unit": "MW",
                     "values": [
-                        39000.0,
-                        39000.0
+                        0.0,
```

Comment:
Intentional?

Reply:
The intention is to ensure that both consumer systems behaves the same. No specific intention in regards to the result values.